lib/HttpClient.ts

```diff
@@ -142,35 +142,35 @@ export class HttpClient {
         }
     }
 
-    public options(requestUrl: string, additionalHeaders?: ifm.IHeaders): Promise<HttpClientResponse> {
+    public options(requestUrl: string, additionalHeaders?: ifm.IHeaders): Promise<ifm.IHttpClientResponse> {
```

Comment:
does this break consumers on a patch update?

Reply:
It shouldn't, I added an interface but the structure is the same as it was before(HttpClientResponse).

Reply:
Once the implementation adds something that the interface doesn't have, it will break consumers. Consider this:

![image](https://user-images.githubusercontent.com/12102068/34857397-906ddd08-f718-11e7-8d4b-f303dfc9e28f.png)

Reply:
I think this is fine because the new major version (1.x) and all it's patches have been in preview.